### PR TITLE
Bootstrap workshop page actions partial

### PR DESCRIPTION
**Pull request by KimberleyCook**
FYI the new buttons look like this

Screenshot 2020-11-20 at 13 50 53

### CODE REVIEW

**Review by matyikriszta — APPROVED** [review]
👍

File: app/views/workshops/_actions.html.haml [hunk 1]
```diff
@@ -15,19 +14,20 @@
 
 - if logged_in? && @workshop.future? && (@workshop.invitable || @workshop.open_for_rsvp?)
   - if @workshop.attendee?(current_user)
-    = button_to "Manage your invitation", rsvp_workshop_path(@workshop), method: :post, class: 'button round '
+    = button_to "Manage your invitation", rsvp_workshop_path(@workshop), method: :post, class: 'btn btn-primary'
   - else
     - if current_user.student? && current_user.coach?
-      = button_to "Attend as a student", rsvp_workshop_path(@workshop, role: "Student"), method: :post,class: 'button round '
-      = button_to "Attend as a coach", rsvp_workshop_path(@workshop, role: "Coach"), method: :post,class: 'button round '
+      %div.d-flex
```
Comment (despo) [inline app/views/workshops/_actions.html.haml:20]: Looks good. We can skip `%div` though as we don't tend to use it across our views unless there is no css class specified as `.class-name` translates by default to `<div class="class-name"></div>`

**Review by despo — APPROVED** [review]
👍

File: app/views/workshops/_actions.html.haml (hunk 1 above)
Comment (KimberleyCook) [reply app/views/workshops/_actions.html.haml:20]: Ah yes of course, nice catch. Will update.

**Pull request merged**